### ISSUE TITLE
[Feature request] Option to unapprove follow request

### ISSUE THREAD

**Issue by christophehenry** [tootsuite/mastodon#6434]
Unless I'm wrong, there's no way to unapprove follow request user by user (and there is even no contact page available on Mastodon). This causes a privacy concern for accounts that were locked after the account was created and started to have followers. I'm suprised nobody raised that point in an issue (unless I didn't search correctly, in which case I apologize).

**Comment by nightpool** [comment]
What do you mean by contact page? What does that have to do with follow
requests?

**Comment by nightpool** [comment]
You can "disapprove follow requests" (remove existing followers) by
blocking users you no longer want to be able to follow you. You can decline
pending follow requests just on the follow request page itself. Does that
answer your question?

**Comment by Cassolotl** [comment]
I think this might be a duplicate of this older issue: Removing remote followers should be possible without blocking/unblocking #5708

**Comment by christophehenry** [comment]
> What do you mean by contact page?

The list of people you follow and follow you.

> I think this might be a duplicate of this older issue: Removing remote followers should be possible without blocking/unblocking #5708

It appears so, indeed.

**Comment by Cassolotl** [comment]
Awesome. If you feel happy to move the discussion over to that thread, closing this one might be helpful, I think?

**Comment by christophehenry** [comment]
> Awesome. If you feel happy to move the discussion over to that thread, closing this one might be helpful, I think?

True.
